kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -121,7 +121,7 @@ public KieServerImpl(KieServerStateRepository stateRepository) {
             logger.info("Starting server in '" + mode.name() + "' mode.");
         } catch (Exception ex) {
             mode = KieServerMode.DEVELOPMENT;
-            logger.warn("Unable to parse value of " + KieServerConstants.KIE_SERVER_MODE + " = " + modeParam + "; supported values are 'REGULAR' or 'DEVELOPMENT'. Falling back to 'DEVELOPMENT' mode.");
+            logger.warn("Unable to parse value of " + KieServerConstants.KIE_SERVER_MODE + " = " + modeParam + "; supported values are 'REGULAR' or 'PRODUCTION'. Falling back to 'DEVELOPMENT' mode.");
```

Comment:
REGULAR' or 'PRODUCTION?

Reply:
@pefernan here the message should be:

logger.warn("Unable to parse value of " + KieServerConstants.KIE_SERVER_MODE + " = " + modeParam + "; supported values are 'DEVELOPMENT' or 'PRODUCTION'. Falling back to 'DEVELOPMENT' mode.");

Reply:
@porcelli I missed this!